File: javascript/abaixo_da_media.js
const n = parseInt(prompt("Quantos elementos vai ter o vetor?"));

let vetor = [], mediaVetor = 0;

for(i = 0; i < n; i++) {
    vetor[i] = parseFloat(prompt("Digite um numero:"));
    mediaVetor += vetor[i];
}

mediaVetor /= n;

console.log("MEDIA DO VETOR = %.3f", mediaVetor);


console.log("ELEMENTOS ABAIXO DA MEDIA:");

for(i = 0; i < n; i++) {
    if (vetor[i] < mediaVetor) {
        console.log("%.1f", vetor[i]);
    }
}

File: python/abaixo_da_media.py
n = int(input("Quantos elementos vai ter o vetor? "))

vetor = []
soma_vetor = 0

for i in range(0, n):
    vetor.append(float(input("Digite um numero: ")))
    soma_vetor += vetor[i]

media_vetor = soma_vetor / n

print(f"MEDIA DO VETOR = {media_vetor:.3f}")

print("ELEMENTOS ABAIXO DA MEDIA:")
for i in range(0, n):
    if vetor[i] < media_vetor:
        print(f"{vetor[i]}")

